docs/index.rst

```diff
@@ -0,0 +1,56 @@
+.. DYAD documentation master file, created by
+   sphinx-quickstart on Thu Jan 26 10:12:01 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+DYAD: Dynamic and Asynchronous Data Streamliner
+===============================================
+
+DYAD is a synchronization and data movement tool for computational science workflows built
+on top of Flux. DYAD aims to provide the benefits of in situ and in transit tools
+(e.g., fine-grained synchronization of consumer applications, fast data access due to spatial locality)
```

Comment:
`of consumer applications` It is the synchronization between producer and consumer, or (read and write)